.travis.yml

```
@@ -5,19 +5,18 @@ os:
   - osx
 
 julia:
-  - 0.6
   - 0.7
   - 1.0
   - nightly
 
 matrix:
   allow_failures:
-    - 1.0
     - nightly
 
 notifications:
   email: false
-# uncomment the following lines to override the default test script
-#script:
-#  - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#  - julia --check-bounds=yes -e 'Pkg.clone(pwd()); Pkg.build("MA97"); Pkg.test("MA97"; coverage=true)'
+
+script:
+ - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+ - julia --check-bounds=yes -e 'using Pkg; Pkg.clone(pwd()); Pkg.build("HSL"); Pkg.test("HSL"; coverage=true)'
+ - julia --check-bounds=yes -e 'using Pkg; Pkg.add("BinDeps"); Pkg.add("Compat"); Pkg.build("BinDeps"); using BinDeps; BinDeps.debug("HSL")'
```

The problem is that `BinDeps` is not working correctly on Julia 1.0. I suggest to comment such dependency test for when this Pkg is working again. 
```suggestion
  # - julia --check-bounds=yes -e 'using Pkg; Pkg.add("BinDeps"); Pkg.add("Compat"); Pkg.build("BinDeps"); using BinDeps; BinDeps.debug("HSL")'
```